asyncscope.md

```diff
@@ -1825,7 +1825,32 @@ spec here
 
 ## `execution::nest()`
 
-spec here
+::: add
+__§34.9.11.14 `std::execution::nest` [exec.nest]__
+
+[1]{.pnum} `nest` nests its input sender within the dynamic lifetime of the async scope referenced by its input async
```

Comment:
> `nest` nests

we'll need to find a way to describe `nest` that doesn't use the word "nest". something along the lines of: 

> `nest` associates a sender with an async scope such that the scope can track the lifetime of any async operations created with the sender.